docker/base-deps/Dockerfile

```diff
@@ -31,9 +30,12 @@ RUN apt-get update -y && apt-get upgrade -y \
         flatbuffers \
         cython==0.29.0 \
         numpy==1.15.4 \
+        psutil \
+        blist \
```

Comment:
No longer required by serve. So it's safe to remove.

Reply:
Fantastic!